cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/index.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import { STEPS } from 'components/PluginJSONCreator/Create/steps';
+import { createContextConnect } from 'components/PluginJSONCreator/Create';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+
+const styles = (): StyleRules => {
+  return {
+    comp: {
+      borderRight: '1px solid #e0e0e0',
```

Comment:
nit: We should probably use the colors from our theme file. `colors.ts/scss` should have this color that we can use from the theme palette.